scripts/docker-compose.yml

```diff
@@ -298,7 +307,7 @@ services:
       - host.docker.internal:host-gateway
 
   endorser-agent-1:
-    image: ghcr.io/hyperledger/aries-cloudagent-python:py3.9-0.10.3
+    image: ghcr.io/hyperledger/aries-cloudagent-python:py3.9-0.10.2
```

Comment:
Should stay .3

Reply:
Sigh... Why does this keep reverting itself.

Reply:
Done.